src/components/Rout.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import {
+  BrowserRouter as Router, Switch, Route, Link,
+} from 'react-router-dom';
+import './Rout.css';
+import Calculator from './Calculator';
+
+export default function Rout() {
+  return (
+    <Router>
+      <nav>
+        <ul>
+          <li>
+            <Link to="/">Home</Link>
+          </li>
+          <li>
+            <Link to="/Calculator">Calculator</Link>
+          </li>
+          <li>
+            <Link to="/Quote">Quote</Link>
+          </li>
+        </ul>
+      </nav>
+      {/* Used switch component to display different content on UI */}
+      <Switch>
+        <Route exact path="/">
+          <Home />
+        </Route>
+        <Route path="/Calculator">
+          <CalculatorPage />
+        </Route>
+        <Route path="/Quote">
+          <Quote />
+        </Route>
+      </Switch>
+    </Router>
+  );
+}
+
+function Home() {
+  const text = 'Welcome to Math Magicians, your one-stop destination for all things math!\n\n'
+    + 'Whether you\'re a student, a teacher, or just someone who loves math, '
+    + 'we\'ve got everything you need to enhance your math skills and knowledge. '
+    + 'From basic arithmetic to advanced calculus, we\'ve got it all covered. '
+    + 'So why wait? Start exploring our site today and take your math skills to '
+    + 'the next level!';
+  return (
+    <div className="Home-content">
+      <h1>Math Magicians</h1>
+      <h2>Welcome to our page!</h2>
+      <p>{text}</p>
+    </div>
+  );
+}
+function CalculatorPage() {
+  return (
+    <div className="calculator-content">
+      <h1>Math Magicians</h1>
+      <Calculator />
```

Comment:
- [ ] [OPTIONAL] As this is not a strict requirement for this milestone and should have been caught earlier, I'm marking it as optional.
There is an error on the console which clearly shows that you have provided a `value` prop to an input field without an `onChange` handler. This input field in question is what you used as your display in the `<Calculator />` component 😄 Please consider changing it to a div instead?
**HINT:**
Something like this won't be too bad 😉 
```javascript
<div className="input">{state.next || state.total || '0'}</div>
```
![image](https://user-images.githubusercontent.com/46839548/221898112-81b79bb3-d7a1-48a8-af23-d431ef1868c0.png)